src/pages/_app.tsx

```diff
@@ -29,7 +29,9 @@ if (
       api_host: 'https://eu.posthog.com',
       // Enable debug mode in development
       loaded: (posthog) => {
-        if (process.env.NODE_ENV === 'development') posthog.debug()
+        posthog.debug(
```

Comment:
so this is emitting the logs? Is Posthog still going to work on staging and production but the debug mode which emits the console logs is going to be deactivated?

Reply:
I thought we wanted to see posthog console logs on the development and barge. Don't we ?

Reply:
disabled completely now